packages/mobx-fog-of-war/src/Store.ts

```diff
@@ -108,7 +113,7 @@ export class Store<Args,Data,Err> {
 
             if(staleTime === -1) return false;
             if(staleTime === 0) return true;
-            return new Date(Date.now()) > new Date(item.time.getTime() + staleTime);
+            return new Date(Date.now()) > new Date(item.time.getTime() + staleTime * 1000);
```

Comment:
Ideally this doesn't happen on get but on an interval. You're going to accumulate a lot of objects that never get used again, since you only discard when you lookup..

Reply:
Ah, yeah I'm deliberately keeping the "should I re-request an item" part separate from the "should I remove an item from cache" feature. The second feature will come later and probably will be on an interval yeah. Right now the cache only gets bigger you're right.

Reply:
Sounds a great approch